File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.java
class Solution {
    public boolean canAliceWin(int[] nums) {
        int sum1,sum2;
        sum1=sum2=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]<10) sum1+=nums[i];
            else sum2+=nums[i];
        }
        return sum1!=sum2;
    }
}

File: 3515-find-if-digit-game-can-be-won/find-if-digit-game-can-be-won.py
class Solution(object):
    def canAliceWin(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        sum1=sum2=0
        for x in nums:
            if x<10: sum1+=x
            else: sum2+=x
        return sum1!=sum2